scripts/dev_scripts/integration_tests.sh

```diff
@@ -19,6 +21,7 @@ UPDATE=0
 if [ $# -eq 3 ] && [ "$3" == "--update" ] ; then
     echo "Updating the expected results to match those currently produced by Macaron."
     UPDATE=1
+    COMPARE_VSA="$COMPARE_VSA --update"
```

Comment:
The `compare_vsa.py` script has a `--update` flag to update the expected VSA payload files. 

On this line, the `--update` flag of the `compare_vsa.py` script is turned on if the `integration_tests.sh` script is run with the `--update` flag.

Reply:
I see, thanks. How about moving it to `check_or_update_expected_output` so that we can consistently use `check_or_update_expected_output` like other tests?

Reply:
I'm not sure how this could be achieved in the `check_or_update_expected_output` function, because the function either runs `python <script> <result_file> <expected_file>` or `cp <result_file> <expected_file>`. When used, this function also comes before the script name, so there is no way to pass the `--update` flag into the `compare_vsa.py` script as the flag must come after the script name.

Reply:
How about checking the script name and passing `--update` instead of running `cp` if the script is `COMPARE_VSA`?

Reply:
Hmm, I guess I could do that, although I'm a bit unsure of whether it is indeed better. I guess it provides a more consistent interface for creating tests, with the trade-off of a bit more complex implementation of the `check_or_update_expected_output` function, and aiming for easier/more convenient test addition/creation is a good idea.

Reply:
See e621ec3a2711205766450727cdd4736d2d70dcb3.